o1vm/src/interpreters/mips/witness.rs

```diff
@@ -1286,7 +1286,10 @@ impl<Fp: Field, PreImageOracle: PreImageOracleT> Env<Fp, PreImageOracle> {
             let insn = self.get_opcode().unwrap();
 
             // Approximate instruction per seconds
-            let how_many_steps = step as usize - start.step;
+            let how_many_steps = (step as usize)
```

Comment:
You still need this even with the latest changes on master? :thinking:

Reply:
Why you think this is redundant? It will be good to have the corresponding cause message should we later hit the issue again, isn't it?

Reply:
I'm just surprised we do need that. I'm fine with the change.

Reply:
I don't think we need it. I though it will be fine addition to leave it. If you don't want it I'll remove it of course.

Reply:
Will be removed